microsoft/testsuites/ltp/ltpsuite.py

```diff
@@ -58,6 +58,7 @@ def verify_ltp_lite(
         tests = variables.get("ltp_test", "")
         skip_tests = variables.get("ltp_skip_test", "")
         ltp_tests_git_tag = variables.get("ltp_tests_git_tag", "")
+        ltp_tar_file = variables.get("ltp_file_path", "")
```

Comment:
the test variable name should be clear, like `ltp_prebuilt_file`. Two names should be consistent.

Reply:
The change is done, thanks